packages/sdks/src/server-index.ts

```diff
@@ -27,3 +27,5 @@ export type { RegisteredComponent } from './context/types.js';
 export type { ComponentInfo } from './types/components.js';
 
 export { fetchBuilderProps } from './functions/fetch-builder-props.js';
+export { isBrowser } from './functions/is-browser.js';
+export { isServer } from './functions/is-server.js';
```

Comment:
I don't think we want to export these in the new SDKs. It makes little sense to provide our users with helpers like these: they have nothing to do with SDK functionality.

What is the use case for adding them here?

Reply:
We have exposed these helpers before in Gen1 and they are widely used in client code, example - in our site. Migrating to gen2 will require users with a substitute, thus exporting them

Reply:
I understand, and that's the way to go. I would rather users rely on their own isBrowser/isServer helpers than reuse ours. The Builder SDK shouldn't be exporting generic helpers such as these: they are unrelated to the Builder SDK's purpose.